apps/stable_diffusion/web/index.py

```diff
@@ -244,7 +245,9 @@ def register_outputgallery_button(button, selectedid, inputs, outputs):
                         upscaler_status,
                     ]
                 )
-            with gr.TabItem(label="DocuChat(Experimental)", id=10):
+            with gr.TabItem(label="DocuChat Upload", id=10):
+                h2ogpt_upload.render()

```

Comment:
If I'm not wrong this just adds a tab for uploading the documents, right? This doesn't create the vector database.

Reply:
I thought the database was built during the call to `evaluate`, but if not I can hop on for a few hours on Saturday and add that.